File: problems/easy/discount/__tests__/discount.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <stdbool.h>
#include "../discount/discount.c"

void test(){
    bool passed = true;
    if (cal_discount(1500, 50)!= 750)
    {
        printf("❌ failed c test 1");
        passed = false;
    }
    if (cal_discount(89, 20)!= 71.2)
    {
        printf("❌ failed c test 1");
        passed = false;

    }
    if (passed)
    {
        printf("✅ passed all c tests");
    }
    
    

}

int main(){
    test();

    return 0;
}

File: problems/easy/discount/__tests__/discount.py
import sys
import os

sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))

from discount import getdiscountedPS

def tests():
    if getdiscountedPS(100, 75) != 25:
        print("❌ failed python tests")
    else:
        print("✅ passed python tests")


if __name__ == "__main__":
    tests()
